src/ui/public/query_bar/components/__snapshots__/query_bar.test.tsx.snap

```diff
@@ -19,10 +19,10 @@ exports[`QueryBar Should disable autoFocus on EuiFieldText when disableAutoFocus
       onOutsideClick={[Function]}
     >
       <div
-        aria-controls="typeahead-items"
+        aria-controls="kbnTypeahead__items"
```

Comment:
@cchaos you've been using kbn for ui/public right?

Reply:
Yeah and anything under `legacy/`